packages/ckeditor5-engine/src/view/placeholder.ts

```diff
@@ -37,10 +38,9 @@ const documentPlaceholders = new WeakMap<Document, Map<Element, PlaceholderConfi
  * editable root elements.
  * @param options.keepOnFocus If set `true`, the placeholder stay visible when the host element is focused.
  */
-export function enablePlaceholder( { view, element, text, isDirectHost = true, keepOnFocus = false }: {
+export function enablePlaceholder( { view, element, isDirectHost = true, keepOnFocus = false }: {
```

Comment:
You have removed `options.text` property, yet it remains in the API docs:

```
* @param options.text Placeholder text.
```

It should be removed, since it's no longer there.